### PR TITLE
docs: add ip support into route host field

### PR DESCRIPTION
**Pull request by starsz**
### What this PR does / why we need it:

Now, we can fill the IP in the route host field to match.
Add doc to introduce it.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/admin-api.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Note: When the `Admin API` is enabled, it will occupy the API prefixed with `/ap
...
-| host             | False, can't be used with `hosts`        | Match Rules | Currently requesting a domain name, such as `foo.com`; PAN domain names such as `*.foo.com` are also supported.                                                                                                                                                                                                                                                                                                                    | "foo.com"                                            |
+| host             | False, can't be used with `hosts`        | Match Rules | Currently requesting a domain name or IP address, such as `foo.com`, `127.0.0.1` ; PAN domain names such as `*.foo.com` are also supported, `*.0.0.1` also.                                                                                                                                                                                                                                                                                                                    | "foo.com"                                            |
```
Comment (juzhiyuan) [inline docs/en/latest/admin-api.md:77]: CIDR?

Comment (starsz) [reply docs/en/latest/admin-api.md:77]: Not CIDR.
CIDR is like: "a.b.c.0/24"
But this is like: "*.0.0.1"
😀

Comment (spacewander) [reply docs/en/latest/admin-api.md:77]: Better not mention this "feature" as it is supported unexpectedly. It just happens to work.

**Review by spacewander — CHANGES_REQUESTED** [review]

**Comment by spacewander** [comment]
The IP address should not be support as the host. If this does something harmful, we should consider rejecting it via validation.

File: docs/en/latest/admin-api.md (hunk 1 above)
Comment (starsz) [reply docs/en/latest/admin-api.md:77]: OK.Got it.

**Comment by juzhiyuan** [comment]
cc @moonming to know this.

**Comment by moonming** [comment]
> The IP address should not be supported as the host. If this does something harmful, we should consider rejecting it via validation.

agreed.  cc @membphis

**Pull request closed without merging**